packages/create-redwood-app/tests/template.test.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-env node, es2022 */
+
+import assert from 'node:assert'
+import fs from 'node:fs'
+import path from 'node:path'
+import { test } from 'node:test'
+import { fileURLToPath } from 'node:url'
+
+const TEMPLATE_PATH = path.resolve(
+  path.dirname(fileURLToPath(import.meta.url)),
+  '../template'
+)
+
+// If you add, move, or remove a file from the create-redwood-app template,
+// you'll have to update this test.
+test("the file structure hasn't unintentionally changed", () => {
+  assert.deepStrictEqual(walk(TEMPLATE_PATH), [
+    '/.editorconfig',
+    '/.env',
+    '/.env.defaults',
+    '/.env.example',
+    '/.nvmrc',
+    '/.vscode/extensions.json',
+    '/.vscode/launch.json',
+    '/.vscode/settings.json',
+    '/.yarn/releases/yarn-3.5.0.cjs',
+    '/.yarnrc.yml',
```

Comment:
Should've seen it coming but this test fails on Windows for reasons you can probably guess. Need to normalize the paths or something similar